docs/docsite/rst/user_guide/playbooks_strategies.rst

```diff
@@ -36,14 +35,114 @@ or pass it on the command line: `ansible-playbook -f 30 my_playbook.yml`.
+          command: hostname
+        - name: second task
+          command: hostname
+
+In the above example, if we had 4 hosts in the group 'webservers', Ansible would execute the play completely (both tasks) on 2 of the hosts before moving on to the next 2 hosts::
+
+
+    PLAY [webservers] ****************************************
+
+    TASK [first task] ****************************************
+    changed: [web2]
+    changed: [web1]
+
+    TASK [second task] ***************************************
+    changed: [web1]
+    changed: [web2]
+
+    PLAY [webservers] ****************************************
+
+    TASK [first task] ****************************************
+    changed: [web3]
+    changed: [web4]
+
+    TASK [second task] ***************************************
+    changed: [web3]
+    changed: [web4]
+
+    PLAY RECAP ***********************************************
+    web1      : ok=2    changed=2    unreachable=0    failed=0
+    web2      : ok=2    changed=2    unreachable=0    failed=0
+    web3      : ok=2    changed=2    unreachable=0    failed=0
+    web4      : ok=2    changed=2    unreachable=0    failed=0
+
+
+You can also specify a percentage with the ``serial`` keyword. Ansible applies the percentage to the total number of hosts in a play to determine the number of hosts per pass::
+
+    ---
+    - name: test play
+      hosts: webservers
+      serial: "30%"
+
+If the number of hosts does not divide equally into the number of passes, the final pass contains the remainder. In this example, if you had 20 hosts in the webservers group, the first batch would contain 6 hosts, the second batch would contain 6 hosts, the third batch would contain 6 hosts, and the last batch would contain 2 hosts.
+
+You can also specify batch sizes as a list. For example::
+
+    ---
+    - name: test play
+      hosts: webservers
+      serial:
+        - 1
+        - 5
+        - 10
+
+In the above example, the first batch would contain a single host, the next would contain 5 hosts, and (if there are any hosts left), every following batch would contain either 10 hosts or all the remaining hosts, if fewer than 10 hosts remained.
+
+You can list multiple batch sizes as percentages::
+
+    ---
+    - name: test play
+      hosts: webservers
+      serial:
+        - "10%"
+        - "20%"
+        - "100%"
+
+You can also mix and match the values::
+
+    ---
+    - name: test play
+      hosts: webservers
+      serial:
+        - 1
+        - 5
+        - "20%"
+
+.. note::
+     No matter how small the percentage, the number of hosts per pass will always be 1 or greater.
+
+Restricting execution with ``throttle``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``throttle`` keyword limits the number of workers up to the maximum set with the forks setting or ``serial``. It can be set at the block and task level. Use ``throttle`` to restrict tasks that may be CPU-intensive or interact with a rate-limiting API::
 
```

Comment:
I'm not following how throttle applies to serial?

Reply:
I'll try to express it more clearly.